src/modules/spaces/actions.js

```diff
@@ -57,6 +58,17 @@ export function fetchById(spaceId) {
         const user_id = value.user_id
         const has_user = !!(users.find(e => e.id === user_id))
         if (!has_user) { dispatch(userActions.fetchById(user_id)) }
+
+        // TODO(matthew): Right now, the space has an embedded user and organization record... why are we doing this
+        // extra fetching?
+        if (value.organization_id) {
```

Comment:
I realize that this wasn't done before, but now that this function is so long, I think i would appreciate splitting it up into a helper like ``needs_organization(value.organization_id, getState().organizations)``